src/jit/codegenlegacy.cpp

```diff
@@ -5646,12 +5646,10 @@ void CodeGen::genCodeForQmark(GenTreePtr tree, regMaskTP destReg, regMaskTP best
             // If any candidates are not alive at the GT_QMARK node, then they
             // need to be spilled
 
-            VARSET_TP VARSET_INIT(compiler, rsLiveNow, compiler->compCurLife);
-            VARSET_TP VARSET_INIT_NOCOPY(rsLiveAfter, compiler->fgUpdateLiveSet(compiler->compCurLife,
-                                                                                compiler->compCurLifeTree, tree));
+            VARSET_VALARG_TP rsLiveNow(compiler->compCurLife);
```

Comment:
But the *intent* is for it to be used in a function signature. (I'm not sure the reason, honestly). In this case, why wouldn't you just use VARSET_TP?